### PR TITLE
SpeedDial Feedback

### PR DESCRIPTION
**Pull request by adriankapka**
Proposed changes to the operation of the feedback, using encoders (tested on AKAI APC40).

In addition, changes the display milliseconds

Please, suggestions or correct

### CODE REVIEW

File: ui/src/virtualconsole/vcspeeddial.cpp [hunk 1]
```diff
@@ -214,7 +214,7 @@ void VCSpeedDial::slotDialTapped()
 void VCSpeedDial::updateFeedback()
 {
     int fbv = (int)SCALE(float(m_dial->value()), float(m_absoluteValueMin),
-                     float(m_absoluteValueMax), float(0), float(UCHAR_MAX));
+                     float(m_absoluteValueMax), float(1), float(UCHAR_MAX));
```
Comment (plugz) [inline ui/src/virtualconsole/vcspeeddial.cpp]: What happens when you want to send ZERO as feedback ?

**Comment by mcallegari** [comment]
As Travis says, this patch doesn't build. Please fix.

**Comment by adriankapka** [comment]
Excuse me, by chance I removed one line too many.It should be OK

**Comment by mcallegari** [comment]
Superseeded by #929

**Pull request closed without merging**